generator.py

```diff
@@ -79,7 +80,7 @@ def regen_crates(path, conf):
     elif path.match("Gir*.toml"):
         print('==> Regenerating "{}"...'.format(path))
 
-        args = [conf.gir_path, "-c", path, "-o", path.parent, "-d", conf.gir_files_path]
+        args = [conf.gir_path, "-c", path, "-o", path.parent, "-d", conf.gir_files_path, "-d", GTK4_GIR_FILES_DIRECTORY]
```

Comment:
That was my initial thought as well, but you could perhaps pick a more general solution from https://gitlab.freedesktop.org/gstreamer/gstreamer-rs/-/merge_requests/719/diffs?commit_id=d6f94214959ea9e5fd2e631d7ab6069961b97e9c.

Unfortunately the name of this secondary directory is now the first difference between gtk/gtk4/gstreamer `generator.py`, perhaps we should find a cleaner solution (and move this file to a more sensible, _shared_ place like `gir-files` itself). CC @sdroege @GuillaumeGomez.

Reply:
I would remove the default for `-d` in `generator.py` and rely on the `Gir.toml` files by default. If `-d` is provided it would be for overriding the `Gir.toml`.